ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -370,27 +420,15 @@ const DisplayIcon = (props: DisplayIconProps) => {
     ],
     [
       'cromwellConfig',
-      () => {
-        const cromwellAssets = appAssets.find(
-          (aa) => aa.appType === UIAppType.CROMWELL
-        );
-        return (
-          <FlexRow
-            style={{
-              height: '100%',
-              alignItems: 'center',
-              justifyContent: 'space-around',
-            }}
-          >
-            <img
-              data-test-id={'help-sidebar-icon-' + icon.id}
-              src={cromwellAssets.icon}
-              alt={icon.label}
-              style={{ ...icon.style, position: 'absolute' }}
-            />
-          </FlexRow>
-        );
-      },
+      () =>
+        displayAppStatusIcon(
+          icon,
+          workspace.namespace,
+          userSuspended,
+          config,
+          userApps?.find((app) => app.appType === AppType.CROMWELL)?.status,
```

Comment:
Minor: There's a lot of cases to cover here (`userApps` is `undefined`, Cromwell doesn't exist), making the entire component complex and testing difficult. Is there a way to extract some of the logic into a helper method or increase the type-safety of these variables? I don't have specific suggestions at the moment, so consider this optional.

Reply:
I think that that is a valid consideration, but I think that since this expression uses optional chaining/elvis operators, this expression will return undefined if userApps is undefined or if Cromwell is not found.